app/controllers/users_controller.rb

```diff
@@ -52,7 +59,7 @@ def update
     if @user.update(user_params) && update_membership(membership_params)
       confirm_change(@user)
       if @current_user.admin_in? @roster
-        redirect_to roster_users_path(@roster)
+        redirect_back fallback_location: roster_users_path(@roster)
```

Comment:
Since I was using the update action to handle activation I didn't want to redirect back to the index while making a change on the inactive page. Although, that page is gone now so I can change it back.